src/pages/en/concepts/mpa-vs-spa.md

```diff
@@ -0,0 +1,90 @@
+---
+layout: ~/layouts/MainLayout.astro
+title: MPAs vs. SPAs
+description: TODO
+i18nReady: true
+---
+
+Understanding the tradeoffs between Multi-Page Application (MPA) and Single-Page Application (SPA) architecture is key to understanding what makes Astro different from other web frameworks like Next.js and Remix.
+
+## Terminology
+
+**A Multi-Page Application (MPA)** is a website consisting of multiple HTML pages, mostly rendered on a server. When you navigate to a new page, your browser requests a new page of HTML from the server. **Astro is an MPA framework.** More traditional MPA frameworks also include Ruby on Rails, Python Django, PHP Laravel, Wordpress, and static site builders like Eleventy, Hugo and VitePress.
+
+**A Single-Page Application (SPA)** is a website consisting of a JavaScript application that loads in the user's browser and then renders HTML locally. SPAs may *also* generate HTML on the server (especially for the the first page load) but SPAs are unique in their ability to render HTML locally when you navigate to a new page. Next.js, Nuxt, SvelteKit, Remix, Gatsby, and Create React App are all examples of SPA frameworks.
+
+## Astro vs. other MPAs
+
+Astro is an MPA framework. However, Astro is unique from other MPA frameworks due to its use of JavaScript as a server runtime. Traditional MPA frameworks would have you write one language on the server (Ruby, PHP, etc.) and JavaScript on the browser. In Astro, you're always writing JavaScript. You can also re-use UI components (like React and Svelte) on both the server and the client. 
+
+The result is that Astro has a developer experience similar to that of Next.js or another modern SPA, even if the user experience is an entirely different MPA site architecture.
+
+## MPAs vs. SPAs
+
+There are three main differences to be aware of when comparing MPAs vs. SPAs:
```

Comment:
```suggestion
There are three main differences between MPAs and SPAs that developers should note:
```
Again, personal preference, but I think the 3 main diffs between is a concise statement, and the being aware of really belongs to WHO those differences matters to.

"that developers should be aware of:"
"that developers choosing a framework should be aware of:"
"that make a difference to developers (choosing a framework):"
"of note to developers choosing a framework for a project:"

Or a different angle entirely:

"Examine these three key differences between MPAs and SPAs before choosing a framework for your next project:"
"Not sure which site architecture to choose for your project? Here are three key differences you should understand:"
"Trying to decide between MPA and SPA? Consider the following three key differences:"